02week/blog/index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Music Blog</title>
+  <link href="css/style.css" rel="stylesheet" type="text/css" />
+  <link href="https://afeld.github.io/emoji-css/emoji.css" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Bungee Shade' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Condiment' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Ewert' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Nosifer' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Oldenburg' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Risque' rel='stylesheet'>
```

Comment:
Remind me to show you a way to add all of those font using a single `<link>`
Also, the more fonts you are bringing the longer your site will take to load. Usually you should have about 2-3 font at the most.